src/app/loadout/fashion/FashionDrawer.tsx

```diff
@@ -483,6 +484,23 @@ function FashionItem({
   );
 }
 
+function FashSocketTooltip({
```

Comment:
Gosh I'd prefer if we spelled out "Fashion"

Reply:
Eh woops, will fix

Reply:
Done